client/my-sites/controller.js

```diff
@@ -162,7 +163,12 @@ export function renderNoVisibleSites( context ) {
 }
 
 function renderSelectedSiteIsDomainOnly( reactContext, selectedSite ) {
-	reactContext.primary = <DomainOnly siteId={ selectedSite.ID } hasNotice={ false } />;
+	reactContext.primary = (
+		<>
+			<PageViewTracker path="/view/:site" title="Domain Only" />
```

Comment:
@tyxla I've moved the `<PageViewTracker>` component up a level as [recommended](https://github.com/Automattic/wp-calypso/pull/66731#discussion_r955731953)!